myhoard/controller.py

```diff
@@ -353,7 +353,7 @@ def switch_to_observe_mode(self):
                 return
             elif self.mode in {self.Mode.active, self.Mode.promote}:
                 # Master (or almost master) cannot become a standby
-                raise ValueError(f"Switch from {self.mode} to observe mode is not allowed")
+                raise ValueError(f"Switch from {self.mode.value} to observe mode is not allowed")
```

Comment:
Why this change ?

Reply:
This is due to a change to Enum.format behaviour (see https://docs.python.org/3/whatsnew/3.11.html):

> Changed [Enum.__format__()](https://docs.python.org/3/library/enum.html#enum.Enum.__format__) (the default for [format()](https://docs.python.org/3/library/functions.html#format), [str.format()](https://docs.python.org/3/library/stdtypes.html#str.format) and [f-string](https://docs.python.org/3/glossary.html#term-f-string)s) of enums with mixed-in types (e.g. [int](https://docs.python.org/3/library/functions.html#int), [str](https://docs.python.org/3/library/stdtypes.html#str)) to also include the class name in the output, not just the member’s key. This matches the existing behavior of [enum.Enum.__str__()](https://docs.python.org/3/library/enum.html#enum.Enum.__str__), returning e.g. 'AnEnum.MEMBER' for an enum AnEnum(str, Enum) instead of just 'MEMBER'.

Reply:
I chose to keep the same error message.

Reply:
This makes sense, but this should be in the commit messages :stuck_out_tongue:

Reply:
Done!